extensions/azure/README.md

```diff
@@ -0,0 +1,21 @@
+# Azure extension for Azure Data Studio
```

Comment:
We should have a more scoped name for the read me.  I'd recommend for now "Azure Resource Explorer" as the name.

Reply:
I'm going with Azure (Core).